src/core/directives/google-map.ts

```diff
@@ -268,6 +285,17 @@ export class SebmGoogleMap implements OnChanges, OnInit {
     }
   }
 
+  private _fitBounds() {
+    if (this.fitBounds != null && !this.usePanning) {
```

Comment:
@SebastianM You're already ensuring that `this.fitBounds != null` in `_updatePosition()`, no need to check again here...